bindings/python/gpt4all/gpt4all.py

```diff
@@ -0,0 +1,70 @@
+"""
```

Comment:
Use `mkdocs` as set up in the `nomic` python client to generate documentation for this library. See the makefile for commands around local dev `mkdocs serve` and deployment to our s3 hosted websites for the docs.

It may honestly be useful to have a single documentation page `docs.nomic.ai` with documentation for all the various bindings. mkdocs templates look nice.